articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,150 @@
+
+1. **Local styles:** Also known as inline. This form is defined within your HTML tags/elements. It is mostly used to style specific elements in your code.
+
+```HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+    </head>
+    <body>
+      <p style = "font-family: sans-serif;
+          font-size: 1.2em
+          font-style: italic;">
+          This paragraph is an example of a local style.
+        </p>
+        <p>This
+    </body>
+  </html>
+```
+
+By using the style tag in the HTML element *p*, we're able to change the font, font-size, and style only for the first *p*. As for the second *p* element it retains its default style.
+
+2. **Page-Level styles:** Page-level styles are defined at the header area of the HTML file. All similar tags, whether elements members of the class or ID selector within the body of the HTML will undergo the changes at once.
+
+```HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+      <meta charset="utf-8">
+      <style type="text/css">
+         body{
+           color: yellow;
+           background-color: red;
+         }
+         p{
+           color: red;
+           background-color: yellow;
+         }
+      </style>
+    </head>
+    <body>
+      <h1>Heading</h1>
+      <p>This paragraph has been styled using page level styling.</p>
+    </body>
+  </html>
+```
+
+This time around we defined the style within the *head* HTML tags. By doing so all elements defined in the style will automatically use the style defined for them. This means that all *p* elements on the page will have a color red and a background-color of yellow. The body on the other hand will have the color yellow and a background-color of red.
+
+3. **External Styles:** The styles used for the webpage are located in a completely different file. The *.HTML* file is linked to the *.css* file that can be imported to modify the style. When developing a website with multiple pages this styling usually comes in handy.
+
+```html
+  <!DOCTYPE html>
+  <html>
+    <head>
+        <meta charset="utf-8">
+        <title>External Styles</title>
+        <link rel="stylesheet" type="text/css" href="css/myStyle.css"/>
+    </head>
+    <body>
+      <h1>Heading</h1>
+        <p>This is an example of External CSS</p>
+    </body>
+  </html>
+```
+
+In this first half of the code there are no style tags defined. However, the styling used for the webpage has been written in another file and linked to this one using the link tags.
+
+```CSS
+  /*This is the CSS file called myStyle.css*/
+  body{
+    background-color: black;
+    color: white;
+  }
+  p{
+    color: purple;
+  }
+```
+
+The CSS written in this file, *mystyle.css*, is used in the code above to implement the design.
+
+### When to use each Style
+With these three styling options in CSS, **the best practice** would be to avoid internal CSS and implement external CSS.
```

Comment:
This sentence seems very vague. Why is it best practice ? 
Why learn internal if we need to avoid it? 
What makes external CSS the best option